arxiv/integration/api/status.py

```diff
@@ -0,0 +1,47 @@
+"""Contants for HTTP status codes. Merely to increase code readability."""
+
+HTTP_100_CONTINUE = 100
```

Comment:
Can we use https://docs.python.org/3/library/http.html#http.HTTPStatus instead?

Reply:
Yes!

Reply:
Python 3 keeps on giving